src/index.css

```diff
@@ -38,6 +38,9 @@
   --selection: #ff7bed35;
   --transparent: #00000000;
 
+  --accent-transparent: #fed8231a;
```

Comment:
Let's use `10` instead of `1a`, 10 = 10%, but you need to check `1a` each time when you want to understand which transparency it is

Reply:
If I set `10`, it will be a different color; instead of that, I want to set